index.js

```diff
@@ -380,3 +389,23 @@ function getX(p) {
 function getY(p) {
     return p.y;
 }
+
+function multiToSingles(multiPointFeature) {
+    const featureTemplate = {
+        'type': 'Feature',
+        'properties': {
+        },
+        'geometry': {
+        }
+    };
+    const newFeatures = [];
+    for (let i = 0;  i < multiPointFeature.geometry.coordinates.length; i++) {
+        const newCoordinates = multiPointFeature.geometry.coordinates[i];
+        const newProperties = multiPointFeature.properties[i];
+        featureTemplate.geometry.properties = newProperties;
+        featureTemplate.geometry.coordinates = newCoordinates;
+        featureTemplate.geometry.type = 'Point';
+        newFeatures.push(featureTemplate);
+    }
```

Comment:
This won't work because JavaScript passes objects by references, so you'll end up with an array of features with exactly the same feature. Additionally, `multiPointFeature.properties` is an array so `[i]` shouldn't be there.